src/common/sdpa_test_iface.cpp

```diff
@@ -32,10 +32,14 @@ dnnl_status_t DNNL_API sdpa_primitive_desc_create(
         bool invert_scale, dnnl_dim_t kv_head_number, bool causal_mask,
         const_dnnl_primitive_attr_t attr, const_dnnl_primitive_attr_t kq_attr,
         const_dnnl_primitive_attr_t vs_attr) {
-    if (auto err = sdpa_attr_check(query_desc, key_desc, value_desc, engine,
-                attr, kq_attr, vs_attr)) {
-        return err;
-    }
+    status_t err = status::success;
+    err = sdpa_desc_check(query_desc, key_desc, value_desc, dst_desc, mask_desc,
+            engine, attr, kq_attr, vs_attr);
+    if (err) { return err; }
+    err = sdpa_attr_check(
+            query_desc, key_desc, value_desc, engine, attr, kq_attr, vs_attr);
+    if (err) { return err; }
```

Comment:
```suggestion
    CHECK(sdpa_desc_check(query_desc, key_desc, value_desc, dst_desc, mask_desc,
            engine, attr, kq_attr, vs_attr));
    CHECK(sdpa_attr_check(
            query_desc, key_desc, value_desc, engine, attr, kq_attr, vs_attr));
```

Reply:
I think we decided to use if statements here because the checks are performed internally so printing the error messages would be redundant. (RE: https://github.com/intel-innersource/libraries.performance.math.onednn/pull/8345#discussion_r1823758156)